omniscidb/QueryBuilder/QueryBuilder.h

```diff
@@ -156,6 +156,20 @@ class BuilderExpr {
   BuilderExpr logicalAnd(const BuilderExpr& rhs) const;
   BuilderExpr logicalOr(const BuilderExpr& rhs) const;
 
+  BuilderExpr bwAnd(const BuilderExpr& rhs) const;
+  BuilderExpr bwAnd(int val) const;
+  BuilderExpr bwAnd(int64_t val) const;
+
+  BuilderExpr bwOr(const BuilderExpr& rhs) const;
+  BuilderExpr bwOr(int val) const;
+  BuilderExpr bwOr(int64_t val) const;
```

Comment:
Should we use explicit bit width everywhere? I'd expect either `int32_t` with `int64_t` or `int`  with `long long`; the latter should be incorrect in our case.

Reply:
`int` and `int64_t` overloads match all other similar functions. We can use `int32_t` and `int64_t` instead, but it should be done globally then, not only for bitwise operations.